src/main/java/radiounit/RadioUnitState.java

```diff
@@ -0,0 +1,37 @@
+/**
+ * 
+ */
+package radiounit;
+
+import common.Carrier;
+import common.FrequencyBand;
+
+/**
+ * @author edavleu
+ *
+ */
+public interface RadioUnitState {
+	
+	public void setup();
+	
+	public void activate();
+	
+	public void deactivate();
+	
+	public void release();
+	
+	public void setupCarrier(Carrier carrier);
+
+	public void signalScaling();
+	
+	public void modifyCarrier(Integer carrierId, FrequencyBand band);
+	
+	public void removeCarrier(Integer carrierId);
+	
+	public void selfDiagnostics();
+	
+	public void removeAllCarriers(Integer carrierId);
+	
+	public String getLabel();
```

Comment:
_getLabel_ was a method name we had made right (and not from documentation)? In that case, would it be beneficial for us to change this to the _toString_ method?